docs/src/manifolds/rotations.md

```diff
@@ -5,11 +5,16 @@ The manifold $\mathrm{SO}(n)$ of orthogonal matrices with determinant $+1$ in $\
 $\mathrm{SO}(n) = \bigl\{R \in \mathbb{R}^{n\times n} \big| RR^{\mathrm{T}} =
 R^{\mathrm{T}}R = \mathrm{I}_n, \det(R) = 1 \bigr\}$
 
- $\mathrm{SO}(n)$ is a subgroup of the orthogonal group $\mathrm{O}(n)$ and also known as the special orthogonal group or the set of rotations group.
+``\mathrm{SO}(n)`` is a subgroup of the orthogonal group $\mathrm{O}(n)$ and also known as the special orthogonal group or the set of rotations group.
```

Comment:
Does this do inline-math? Until now I always preferred `$...$` for inline math (and for a long time `\[...\]` for display) since it's compatible with LaTeX (even the nicer `\(...\)` works with MathJax. So if we change this, we should change it globally?

Reply:
Yes, this does. I use it here because beginning a line with `$` makes Documenter think the equation should stand alone on its own line, which isn't what we want here. The double backtick syntax, which usually behaves just like `$`, does the right thing here. Maybe this is a Documenter bug, but this is a workaround. I also prefer the `$...$` in general.

Reply:
I would prefer then to rephrase this to “The manifold $\mathrm{SO}(n)$” (or the Lie group), since starting a sentence with a formula is not that nice anyways.